Assets/HoloToolkit/Input/Scripts/GazeManager.cs

```diff
@@ -129,5 +129,17 @@ private void UpdateStabilizationPlane()
                 HolographicSettings.SetFocusPointForFrame(newFocusPointPosition, -gazeDirection);
             }
         }
+
+        /// <summary>
+        /// Sends message to the object currently focused on.
+        /// </summary>
+        /// <param name="message">Message to send</param>
+        public void SendMessageToFocusedObject(string message)
```

Comment:
The problem is that the object field in Responses array is populated only statically.

How would OnGazeEnter and OnGazeExit be able to help? They only give you events that the object starts or ends to be gazed on, but these methods don't give you an ability to send custom messages to the focused object.

This is meant to be used from KeywordManager to send messages to focused object: http://i.imgur.com/PAGXMTu.png